clone.js

```diff
@@ -6,7 +6,7 @@ const readline = require('readline').createInterface({
 const path = require('path');
 const fs = require('fs');
 
-readline.question(`Wat is de naam van de nieuwe webcomponent? `, (naam) => {
+readline.question(`Wat is de naam van de nieuwe webcomponent (prefix webcomponent-vl-ui wordt automatisch toegevoegd)? `, (naam) => {
```

Comment:
Detail, maar prefix `webcomponent-vl-ui-` wordt automatisch toegevoegd is misschien vollediger, ik heb het aangepast.